java/org/apache/coyote/AbstractProtocol.java

```diff
@@ -262,6 +267,18 @@ public void setConnectionLinger(int connectionLinger) {
         endpoint.setConnectionLinger(connectionLinger);
     }
 
+    // ------------------------------------------------ HTTP specific properties
+    // ----------------------------------------- queried by StandardContextValve
+    public void setContinueHandlingResponsePolicy(String value) {
+        continueHandlingResponsePolicy = Enum.valueOf(ContinueHandlingResponsePolicy.class, value);
+    }
+
+
+    //todo(malay) move to AbstractProtocol
```

Comment:
What does `malay` mean? Like Malay language?

Reply:
sorry, that was a note to myself that I missed removing